### PR TITLE
Add Citation

### LINKED ISSUE #70: Add CITATION
As @dhomeier suggested in https://github.com/astropy/astropy.github.com/pull/490#issuecomment-1122774660 , please add a CITATION file with appropriate citation. Thanks!

### PR DESCRIPTION
**Pull request by WilliamJamieson**
Add a citation file to resolve #70.

### CODE REVIEW

**Review by pllim — COMMENTED** [review]
As long as asdf-astropy devs are ok with this, it is fine by me. Just a minor comment about change log from me. Thanks!

I also cc the authors mentioned in the paper here, just in case: @perrygreenfield @mdboom @embray

File: CHANGES.rst [hunk 1]
```diff
@@ -2,6 +2,7 @@
 ------------------
 
 - Add converter for the new ``Schechter1D`` model. [#67]
+- Add CITATION file. [#71]
```
Comment (pllim) [inline CHANGES.rst:5]: Not sure if this needs a change log.

(p.s. Not sure why it didn't go with the review submission.)

**Comment by pllim** [comment]
So, what is the plan to get this merged?

File: CITATION [hunk 2]
```diff
@@ -0,0 +1,19 @@
+If you use asdf-astropy for work/research presented in a publication
+(whether directly, as a dependency to anther package), we recommend and
```
Comment (nden) [inline CITATION:2]: typo anther --> another

**Pull request merged**